### ISSUE TITLE
Something equivalent to @ApiParam(allowableValues="one,two")

### ISSUE THREAD

**Issue by xverges** [teamcarma/swagger-jaxrs-doclet#89]
http://docs.swagger.io/swagger-core/v1.3.12/apidocs/com/wordnik/swagger/annotations/ApiParam.html#allowableValues%28%29 results in a parameter defined with the `enum` attribute 
```
{
    "name": "paramName",
    "required": false,
    "type": "string",
    "paramType": "path",
    "allowMultiple": false,
    "enum": [
        "one",
        "two"
    ]
}
```
that results in a nice swagger-ui dropdown. 

Can something like this be built using javadoc tags?

**Comment by conorroche** [comment]
Currently only java enums are translated to allowable values, I will add support for javadoc tags that can do the same

**Comment by xverges** [comment]
Great news! Thanks!

**Comment by conorroche** [comment]
I have now added support for this using the tag @paramsAllowableValues

here is an example:

/**
	 * @paramsAllowableValues p1 A p2 B p1 C
	 * @paramsDefaultValue p1 C
	 */
	@PUT
	@Path("/4")
	public void putEnum(@QueryParam("p1") String p1, @QueryParam("p2") String p2) {
		// noop
	}

**Comment by conorroche** [comment]
closing as fixed for 1.0.5 release
